### ISSUE TITLE
Tile animation

### ISSUE THREAD

**Issue by FXRoux** [atompacman/Xerof#17]
Alternate between two texture two give the effect of wind on the biggest texture resolution

**Comment by atompacman** [comment]
I agree on that (and issue #20), but maybe only **two** sprites will look a little bit _laggy_ so I'd suggest we draw a minimum of 3.

**Comment by FXRoux** [comment]
alright, and since we intend not to turn the texture we should draw multiple tile to pick from, some of them not animated
